### PR TITLE
IZE-222 directory structure for global tunnel configuration

### PR DESCRIPTION
**Pull request by psihachina**
## Changelog:
- add an env dir assignment
- set env dir as the executable dir

#### Global config
[asciicast](https://asciinema.org/a/EHgC6J70znzQCuyWZuPBv0QnF)

#### Local config
[asciicast](https://asciinema.org/a/yerN2TYgviz0h7Ww1fMncz5IS)

### CODE REVIEW

**Review by kobrikx — APPROVED** [review]
lgtm

**Review by AutomationD — COMMENTED** [review]
Looks good, but I would prefer not to show a warning if the local config is not found, maybe better say that `Config in ~/.ize/... is used`?

**Comment by psihachina** [comment]
> Looks good, but I would prefer not to show a warning if the local config is not found, maybe better say that `Config in ~/.ize/... is used`?

Maybe, but I'd rather change it not here.
By the way, this should have been noticed in the previous PR.
Also, I think it's worth discussing because we have cases where we don't use the configuration file.

**Review by AutomationD — APPROVED** [review]
Looks good.

File: internal/config/config.go [hunk 1]
```diff
@@ -18,7 +18,10 @@ import (
 	"github.com/spf13/viper"
 )
 
-const Filename = "ize.hcl"
+const (
+	Filename    = "ize.hcl"
+	defaultPerm = 0774
```
Comment (AutomationD) [inline internal/config/config.go:23]: I would not use 77* here as the file should not be an executable. maybe 665

**Review by AutomationD — CHANGES_REQUESTED** [review]
Looks good, let's set a more reasonable perm for the file.

**Pull request merged**